x/ibc/light-clients/07-tendermint/types/proposal_handle_test.go

```diff
@@ -11,182 +13,248 @@ var (
 	}{
 		{
 			name:                         "not allowed to be updated, not frozen or expired",
 			AllowUpdateAfterExpiry:       false,
 			AllowUpdateAfterMisbehaviour: false,
 			FreezeClient:                 false,
 			ExpireClient:                 false,
-			expPassUnfreeze:              false,
-			expPassUnexpire:              false,
+			expPass:                      false,
 		},
 		{
 			name:                         "not allowed to be updated, client is frozen",
 			AllowUpdateAfterExpiry:       false,
 			AllowUpdateAfterMisbehaviour: false,
 			FreezeClient:                 true,
 			ExpireClient:                 false,
-			expPassUnfreeze:              false,
-			expPassUnexpire:              false,
+			expPass:                      false,
 		},
 		{
 			name:                         "not allowed to be updated, client is expired",
 			AllowUpdateAfterExpiry:       false,
 			AllowUpdateAfterMisbehaviour: false,
 			FreezeClient:                 false,
 			ExpireClient:                 true,
-			expPassUnfreeze:              false,
-			expPassUnexpire:              false,
+			expPass:                      false,
 		},
 		{
 			name:                         "not allowed to be updated, client is frozen and expired",
 			AllowUpdateAfterExpiry:       false,
 			AllowUpdateAfterMisbehaviour: false,
 			FreezeClient:                 true,
 			ExpireClient:                 true,
-			expPassUnfreeze:              false,
-			expPassUnexpire:              false,
+			expPass:                      false,
 		},
 		{
 			name:                         "allowed to be updated only after misbehaviour, not frozen or expired",
 			AllowUpdateAfterExpiry:       false,
 			AllowUpdateAfterMisbehaviour: true,
 			FreezeClient:                 false,
 			ExpireClient:                 false,
-			expPassUnfreeze:              false,
-			expPassUnexpire:              false,
+			expPass:                      false,
 		},
 		{
 			name:                         "PASS: allowed to be updated only after misbehaviour, client is frozen",
 			AllowUpdateAfterExpiry:       false,
 			AllowUpdateAfterMisbehaviour: true,
 			FreezeClient:                 true,
 			ExpireClient:                 false,
-			expPassUnfreeze:              true,
-			expPassUnexpire:              false,
+			expPass:                      true,
 		},
 		{
 			name:                         "allowed to be updated only after misbehaviour, client is expired",
 			AllowUpdateAfterExpiry:       false,
 			AllowUpdateAfterMisbehaviour: true,
 			FreezeClient:                 false,
 			ExpireClient:                 true,
-			expPassUnfreeze:              false,
-			expPassUnexpire:              false,
+			expPass:                      false,
 		},
 		{
-			name:                         "allowed to be updated only after misbehaviour, client is frozen and expired",
+			name:                         "PASS: allowed to be updated only after misbehaviour, client is frozen and expired",
```

Comment:
nit: sort by expPass value

Reply:
I decided against this. It becomes harder to to verify that all the test cases are accounted for (or at least I'd need to construct a new method for organizing based on pass that also logically follows the parameter setting). 

Rn it is sorted by the parameters, this makes it easy to verify if there are any missing test cases